nipype/interfaces/ants/registration.py

```diff
@@ -1350,3 +1351,81 @@ def aggregate_outputs(self, runtime=None, needed_outputs=None):
         stdout = runtime.stdout.split('\n')
         outputs.similarity = float(stdout[0])
         return outputs
+
+
+class RegistrationSynQuickInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(3, 2, argstr='-d %d',
+                            usedefault=True, desc='image dimension (2 or 3)')
+    fixed_image = InputMultiPath(File(exists=True), mandatory=True, argstr='-f %s',
+                                 desc='Fixed image or source image or reference image')
+    moving_image = InputMultiPath(File(exists=True), mandatory=True, argstr='-m %s',
+                                  desc='Moving image or target image')
+    output_prefix = Str("transform", usedefault=True, argstr='-o %s',
+                        desc="A prefix that is prepended to all output files")
+
+    # todo ANTSCommandInputSpec already has this, but I can't figure out how to set it without defining it again
+    num_threads = traits.Int(default_value=1, desc='Number of threads (default = 1)', argstr='-n %d')
```

Comment:
you should check if the script checks for the environment variables that is set by the base class instead of this argument. if so the underlying commands in the script will use it. 

but since this script has an explicit flag for threads, you should over-ride the base class.

Reply:
@satra - don't understand how this should be changed. Should `num_threads` be in `RegistrationSynQuickInputSpec` (as it is now), or over-ride somewhere else?

In [this script](https://github.com/ANTsX/ANTs/blob/master/Scripts/antsRegistrationSyNQuick.sh) the default value is `-n 1`, so that's why probably the author of changes used `default_value=1.

Reply:
it should be in:

> `RegistrationSynQuickInputSpec` (as it is now)

since the script explicitly overrides this: https://github.com/ANTsX/ANTs/blob/master/Scripts/antsRegistrationSyNQuick.sh#L388